src/index.html

```diff
@@ -7,6 +7,41 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+   
+    <div class="calendar">
+      <div class="calendar__field calendar__field--mounth-length--31">
+        <div class="calendar__cell calendar__cell--content-1 calendar__cell--start-day--mon"></div>
+        <div class="calendar__cell calendar__cell--content-2"></div>
```

Comment:
```suggestion
    <div class="calendar calendar__month-length--31 calendar__start-day--mon">
        <div class="calendar__cell"></div>
        <div class="calendar__cell"></div>
```
no need additional wrappers. set and style only modifiers for calendar.
no need modifiers for calendar__cell element